src/lib.rs

```diff
@@ -294,6 +299,15 @@ impl Client {
         self
     }
 
+    /// Specifies the struct validation mode that will be used when calling
+    /// [`query::Query::fetch`], [`query::Query::fetch_one`], [`query::Query::fetch_all`],
+    /// and [`query::Query::fetch_optional`] methods.
```

Comment:
... and some new in the future. I think it's better to use common words here and then describe these checks in all those methods (so anyone who calls these methods will see this info in IDE), increasing the chance that ppl will pay attention